exporter/diagnostic_data_collector.go

```diff
@@ -117,7 +117,7 @@ func (d *diagnosticDataCollector) collect(ch chan<- prometheus.Metric) {
 		debugResult(logger, m)
 
 		// MongoDB 8.0 splits the diagnostic data into multiple blocks, so we need to merge them
-		if d.buildInfo.VersionArray[0] >= 8 { //nolint:gomnd
+		if _, ok := m["common"]; ok { //nolint:gomnd,mng
```

Comment:
```suggestion
		if _, ok := m["common"]; ok { //nolint:gomnd,mnd
```

Reply:
Even better :)

Reply:
thanks, it was stupid from my side. I realized, that I don't use magic number there anymore, so I can drop that nolint